RubeesEat.IntegrationTests/CreateTestDb.sql

```diff
@@ -7,7 +7,8 @@ CREATE TABLE Persons
     PersonID VARCHAR(64) PRIMARY KEY,
     FirstName VARCHAR(64) NOT NULL,
     LastName VARCHAR(64) NOT NULL,
-    IsActive BOOLEAN NOT NULL DEFAULT TRUE
+    IsActive BOOLEAN NOT NULL DEFAULT TRUE,
+    LoginName VARCHAR(64)
```

Comment:
We won't change this now but there's a general best practice to not use power-of-two length values in data design. No human understands why a name may only have 63 or 64 characters. Every human understands why it's 50 or a 100 characters.

PS: yes, I'm aware about the meaning of 'human' ;)